README.md

```diff
@@ -138,11 +138,28 @@ The option if partials should be generated if they do not already exist.
 
 #### `cachedir`
 
-Type: `String`
+Type: `String``  
```

Comment:
@mummybot There is an extra ` (tick) on line 141 that [causes the README formatting to be wrong](https://github.com/mummybot/postcss-partial-import/blob/addDependencyTo/README.md#cachedir). There is also extra whitespace at the end of that line.